mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb

```diff
@@ -36,6 +36,13 @@ func resourceStorageBucket() *schema.Resource {
 			customdiff.ForceNewIfChange("retention_policy.0.is_locked", isPolicyLocked),
 		),
 
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(4 * time.Minute),
+			Update: schema.DefaultTimeout(4 * time.Minute),
+			Read:   schema.DefaultTimeout(4 * time.Minute),
+			Delete: schema.DefaultTimeout(4 * time.Minute),
```

Comment:
After taking a closer look at the delete function, it doesn't look like a Timeout is used in its implementation. Sorry about that.
```suggestion
```

Reply:
No problem, removed!